src/bindings/swig/python2/tests/testpy2_key.py

```diff
@@ -155,5 +167,15 @@ def test_iterator(self):
 		self.assertEqual(iter(k).value(),     "user")
 		self.assertEqual(reversed(k).value(), "c")
 
+	def test_helpers(self):
+            k = kdb.Key("user/noarray")
+            self.assertEqual(k.array_elements().index,    False)
+            self.assertEqual(k.array_elements().name,     None)
+            self.assertEqual(k.array_elements().basename, None)
+            k = kdb.Key("user/some/array/#2")
+            self.assertEqual(k.array_elements().index,    2)
```

Comment:
Why is array_elements in plural? It is the second array element? Maybe simply `.array()`?

Reply:
Because it returns the elements which makes the key an array entry. But in general I don't care. I don't use it. I'm just more of a descriptive guy who likes self explaining function names. `.array()` sounds like its converting the key to an array.

Reply:
Then lets leave it as is for now.